ja/translation.json

```diff
@@ -0,0 +1,386 @@
+  "fee": "料金",
+  "l2_info": "L2情報",
+  "unique_identifier_of_block": "メッセージが処理されるブロックの一意の識別子",
+  "block_hash": "ブロックハッシュ",
+  "this_message_does_not": "このメッセージには L2 情報がありません。",
+  "unique_identifier_of_transaction": "メッセージが処理される取引の一意の識別子",
+  "transaction_hash": "取引ハッシュ",
+  "the_message_does_not_have": "メッセージには L2 情報がありません",
+  "time_at_that_the": "メッセージが処理された時刻",
+  "timestamp": "タイムスタンプ",
+  "deposit_initialized": "入金が初期化されました",
+  "deposit_completed": "入金が完了しました",
+  "withdrawal_initialized": "出金が初期化されました",
+  "withdrawal_completed": "出金が完了しました",
+  "accepted": "受け入れられました",
+  "l1_l2_tx_hash": "L1/L2 取引ハッシュ",
+  "l1_contract_address": "L1 コントラクトアドレス",
+  "l2_contract_address": "L2 コントラクトアドレス",
+  "amount": "量",
+  "token": "トークン",
+  "message": "メッセージ",
+  "pending": "ペンディング",
+  "unable_to_locate_event": "このイベントは見つかりませんでした。 まだ受信していないか、選択したネットワークを変更してください。",
+  "input": "入力",
+  "data": "データ",
+  "event_id_block_number": "イベント ID: ブロック番号、取引 インデックス、イベント インデックス",
+  "unique_number_of_block": "イベントが処理されるブロックの一意の番号",
+  "unique_identifer_of_transaction": "イベントが処理される取引の一意の識別子",
+  "sending_party_of_even": "イベントの送信者。",
+  "class_hash_in_that": "イベントが宣言されたクラス",
+  "abi_verified": "ABI 検証済み",
+  "status_of_abi_verification": "ABI 検証のステータス。",
+  "verified": "検証済み",
+  "not_verified": "検証されていない",
+  "danger": "危険",
+  "data_in_the_event": "イベントデータ",
+  "decoded": "デコードされた",
+  "raw": "Raw",
+  "hex": "Hex",
+  "dec": "Dec",
+  "text": "テキスト",
+  "keys": "キー",
+  "keys_in_the_event": "イベントキー",
+  "event": "イベント",
+  "unable_to_locate_class": "このクラスは見つかりませんでした。 まだ宣言されていないか、選択したネットワークを変更してください。",
+  "class": "クラス",
+  "source_code_status": "ソースコードのステータス",
+  "verified_date": "検証日",
+  "created_on": "作成日",
+  "overview": "概要",
+  "name_given_to_class": "検証中にクラスに指定された名前、またはコミュニティによって割り当てられた名前",
+  "transaction_in_which_class": "クラスが宣言された取引",
+  "time_at_that_class": "クラスが宣言された時刻",
+  "license_under_that_verification": "検証が行われたライセンス。",
+  "license": "ライセンス",
+  "compiler_that_was_used": "このクラスのコンパイルとデプロイに使用されたコンパイラー。",
+  "date_on_which_contract": "コントラクトが検証された日。",
+  "verification_date": "検証日",
+  "filename_of_verified_class": "検証されたクラスのファイル名",
+  "filename": "ファイル名",
+  "function_that_allowed_you": "状態を読み取ることができる関数。",
+  "read_functions": "Read 関数",
+  "functions_that_alter_state": "状態を変更できる関数。",
+  "write_functions": "Write 関数",
+  "source_code_of_class": "クラスのソースコード",
+  "source_code": "ソースコード",
+  "class_source_code": "クラスソースコード",
+  "class_application_binary_interface": "クラス アプリケーション バイナリ インターフェイスは、どのバイナリ形式の情報を渡す必要がある関数の呼び出し方法を決定します。",
+  "abi": "ABI",
+  "compiler_version_of_code": "コントラクトソースコードのコンパイル済みバージョン。",
+  "bytecode": "バイトコード",
+  "decompiler_version_of_code": "コントラクトソースコードのデコンパイルバージョン。",
+  "decompiled_code": "デコンパイルコード",
+  "decompiled": "デコンパイルされた",
+  "disassembled": "逆アセンブルされた",
+  "graph": "グラフ",
+  "attribution_this_uses": "援用",
+  "thoth": "Thoth",
+  "the_cairo_bytecode_analyser": "Cairo/Starknet バイトコード アナライザー、逆アセンブラー、逆コンパイラー by",
```

Comment:
So this is where translation becomes harder for me, it was suppose to say in english:
`Cairo/Starknet bytecode analyzer, disassembler, decompiler by`.

And then another part would be `FuzzingLabs`. But since our frontend styling separates them, it becomes harder to do it in Japanese grammar as I think I would have gone with "FuzzingLabs による Cairo/Starknet バイトコード アナライザー、逆アセンブラー、および逆コンパイラー"